.github/workflows/sync-config-schema.yaml

```diff
@@ -72,13 +72,28 @@ jobs:
         with:
           go-version-file: go.mod
 
-      - name: Update main docs version on beta or rc versions
-        # update docs "main" version only on beta or -rc
+      - name: Update docs version on beta or rc versions
+        # update docs - "main" for beta, versioned folder for RC
         if: ${{ steps.release.outputs.is_stable_version == 'false' && steps.release.outputs.is_alpha_version == 'false' }}
         env:
           GH_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
           RELEASE_TAG: ${{ steps.release.outputs.release_tag }}
         run: |
+          # Extract minor version from release tag (e.g., v0.26.1-rc.4 -> 0.26.0)
```

Comment:
Maybe https://github.com/loft-sh/github-actions/tree/main/.github/actions/semver-validation would simplify this logic. It can parse the version and give you some pieces in parsed_version object:

```
{
  "major": 1,
  "minor": 2,
  "patch": 3,
  "prerelease": "alpha.1",
  "build": "build.123",
  "raw": "1.2.3-alpha.1+build.123"
}
```